firmware/hw_layer/adc/adc_inputs.cpp

```diff
@@ -398,6 +398,7 @@ class SlowAdcController : public PeriodicController<256> {
 };
 
 void addChannel(const char *name, adc_channel_e setting, adc_channel_mode_e mode) {
+	(void)name;
```

Comment:
why not just remove the parameter?

Reply:
it is used in many places. not sure about long term plans. may be we will have adc users list like we have for gpio...